tasks/accept-a-maximum-number-of-orders-per-day.json

```diff
@@ -1 +1,20 @@
+  "docs": "This task works by setting your inventory to zero when the order limit is reached. (Specifically, this means setting inventory levels to 0 for all items that have a greater-than-zero inventory level.) There are no popups, or any specific messaging - your inventory will simply be dropped to zero, and if your shop is configured to stop selling out-of-stock products, your customers will be prevented from making additional purchases.\n\nOptionally, this task can restore inventory to its original levels at midnight the next day, or on demand. (Restore levels on demand by enabling this option, then using the \"Run task\" button.). You may also choose to prepend an HTML string to the product description for each product whose inventory levels are dropped to zero by this task. The original product description will be restored whenever the inventory levels are restored by this task. \n\n### Important notes\n\n* ​This task only works for products that have inventory tracking enabled, which are configured to be unavailable when out of stock. This means that the \"Track quantity\" product option needs to be enabled, and the \"Continue selling when out of stock\" option needs to be disabled, for all products.\n* **This task may be not behave as intended if you receive multiple orders per minute. We don't recommend using it for high-volume stores.**",
+  "halt_action_run_sequence_on_error": false,
+  "liquid_profiling": false,
+  "online_store_javascript": null,
+  "order_status_javascript": null,
+  "perform_action_runs_in_sequence": false,
+  "shopify_api_version": "2020-10"
```

Comment:
Please remove this line (and the comma before it, to keep the JSON valid). The task library doesn't currently contain any opinions on API version, and I'm keeping it that way for the moment. Will think on how to make this easy to work with, for contributors.

Reply:
Removed